src/sdk/validators.ts

```diff
@@ -3,8 +3,10 @@ import { z } from 'zod';
 
 export const date = () =>
   z
-    .string()
-    .datetime({ offset: true })
+    .union([
+      z.string().datetime({ offset: true }),
+      z.string().regex(/^\d{4}-\d{2}-\d{2}$/),
```

Comment:
is the result of this regex always a valid input to the `Date` constructor?

Reply:
ahh...it's not, but I mean in the case we're using it I feel like this should be fine, especially since we're using this on specific fields that we know should be dates